src/main/java/org/myrobotlab/swing/InMoovGui.java

```diff
@@ -78,6 +80,11 @@ public InMoovGui(final String boundServiceName, final SwingGui myService) {
     ImageIcon generalIcon = Util.getImageIcon("InMoov.png");
     inmoovPane.addTab("General", generalIcon, generalPanel);
     generalPanel.setLayout(new GridLayout(3, 2, 0, 0));
+    lblNewLabel_1.setForeground(Color.RED);
+    
+    generalPanel.add(lblNewLabel_1);

```

Comment:
If these are only local to this method the labels should not be added as members to the class.